Makefile

```diff
@@ -75,7 +75,7 @@ unit-tests: $(testdata_objects)
 	DISABLE_ROOT_TESTS=$(DISABLE_ROOT_TESTS) go test -short ./... $(EXTRAGOARGS)
 
 all-tests: $(testdata_objects)
-	DISABLE_ROOT_TESTS=$(DISABLE_ROOT_TESTS) go test ./... $(EXTRAGOARGS)
+	DISABLE_ROOT_TESTS=$(DISABLE_ROOT_TESTS) go test -v ./... $(EXTRAGOARGS)
```

Comment:
nit: callers of `make all-tests` sholud set `-v` in `EXTRAGOARGS`

Reply:
Noted and removed `-v` in `EXTRAGOARGS`